packages/core/src/components/dialog/dialog.md

```diff
@@ -35,3 +42,18 @@ whenever possible, as they automatically generate some of this markup.
 More examples of dialog content are shown below.
 
 @css dialog
+
+@## Multistep Dialog
+
+@reactExample MultistepDialogExample
+
+`MultistepDialog` is a wrapper around `Dialog` that maps each sequential step to a view provided by the developer.
+
+The children you provide to this component are rendered as contents inside the
+`Classes.DIALOG` element. Typically, you will want to render a panel with
+`Classes.DIALOG_BODY` that contains the body content for each step.
```

Comment:
I added info under line 54 but it's not showing on the PR for some reason. It can be found `https://github.com/ycvfu/blueprint/blob/vf/multistep-dialog/packages/core/src/components/dialog/dialog.md`.